flink-staging/flink-streaming/flink-streaming-connectors/src/main/java/org/apache/flink/streaming/connectors/kafka/api/simple/SimpleKafkaSource.java

```diff
@@ -71,7 +67,7 @@ public void cancel() {
 
 	@Override
 	public void open(Configuration config) throws InterruptedException {
-		initializeConnection();
+//		initializeConnection();
```

Comment:
Why is this commented out? Eventually we should delete the Kafka connectors using the simple API, there is not much point to keeping them at the current point.